Src/WitsmlExplorer.Frontend/components/AssetsLoader.ts

```diff
@@ -0,0 +1,6 @@
+export class AssetsLoader {
+  static getAssetsRoot() {
```

Comment:
Any warnings added by github should be fixed. This one is an easy fix, just add `: string` as return type

Reply:
Taken care with latest commit